.editorconfig

```
@@ -0,0 +1,8 @@
+root = true
+
+[*.{rb}]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = false
```

I vote: trim it! Trailing whitespace must die!

```suggestion
trim_trailing_whitespace = true
```